### PR TITLE
Rs 2022 06 01

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-06-03-133319_1000x988_scrot

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 1]
```diff
@@ -6,17 +6,17 @@ We face each other in the moonlit corridor.
...
 -- Page 10006, Offset 32820.
 -- ロアに動きはない。
-Roa stands motionless.
+Roa shows no sign of making the first move.
 }
 [sha:17767f2500497636aeea738c46a2c2f6217d3a7c]{
 -- Page 10006, Offset 32821.
 -- 今まで黙って見ていたのは胸くそ悪い余裕の表れだろう。
-The fact that I stood by and watched quietly all this time must be a sign of my disgusting level of tolerance.
+The fact that he just stood by and quietly watched this whole time suggests he's sickeningly cocksure.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:14]: not sure about "cocksure". Maybe "cocky" or just the simple "overconfident"

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 2]
```diff
@@ -41,37 +42,39 @@ Well, it could also be that you just don't know when to give up."
...
 -- Page 408, Offset 32829.
 -- 「死から帰ってきたものは、死というものを理解できる。
-"Only through returning from death can one truly understand it.
+"Those who have returned from death are the only ones who can truly understand it.
 }
 [sha:5d6fc1708b7c8c21bcbea930dff0cccb84b4dccc]{
 -- Page 408, Offset 32830.
 -- 私と君は、その中でもさらに特別な能力があったケースだ。
-Even among them, you and I are such cases who hold a special ability.
+A fraction of those people, such as you and I, even come to possess special abilities.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:61]: drop "even"

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 3]
```diff
@@ -321,52 +353,61 @@ In other words,
...
-"...You're saying that we should be partners?"
+"...You're saying that I should ally myself with you?"
 }
 [sha:7f508cc3a5e44fe0bc56e66f5bc8055955bf0f13]{
 -- Page 425, Offset 32887.
 -- 「まさか。仲間にしてやるんだよ、この私がな。君の意思なぞ知らない。そんな不純物はここで消す。
-"Not at all. I'm the one who will make you my partner. I don't know what your intentions are. So I'll erase those unnecessary things right here.
+"Not at all.
+ I'm saying that I will %{i}make%{/i} you my partner.
+ Your own will doesn't even enter into it.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:381]: Your will doesn't even enter into the equation.

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 4]
```diff
@@ -386,18 +427,18 @@ The areas that represent the source of life only exist in living things."
...
 -- ナイフを握る。
-I clench my knife.
+I grip my knife.
 }
 [sha:e22394029a2733626e565f7bec06f3910a5545c7]{
 -- Page 429, Offset 32897.
 -- <指先|まったん>から<心臓|ちゅうしん>まで生命として<凍結|ていし>する。
 // uh huh...... cool line...
-My life freezes from my fingertips to my heart.
+All vital activity, from my extremities all the way to my core, comes to a standstill.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:436]: make it "vital activities" and "come". "comes" sounds really weird seeing as the preceding sub-clause is a list but I see that you're matching "activity".

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 5]
```diff
@@ -473,27 +517,30 @@ Forgotten the meaning of being human.
...
 -- ヤツの足が止まる。その顔が凍り付く。
-The man's legs stop. His face freezes in place.
+The man comes to a halt. His expression freezes.
 }
 [sha:e90515b0c7750554385e65a027b1391d366dfbec]{
 -- Page 434, Offset 32914.
 -- “生きる”という根拠を失った肉体が、胎児のようにこわばっている。
-His body that has lost its foundation of 'living' becomes stiff like a fetus.
+// Wtf kind of simile is this
+// Is this supposed to mean like a defensive curl?
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:526]: I have literally no idea what this means. "胎児のようにこわばっている" is not a phrase that makes a lick of sense. This is so nonsensical that I honestly suggest asking petrikow

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 6]
```diff
@@ -503,12 +550,12 @@ Under his scope of awareness, there is nothing he can see that should frighten h
 [sha:6174580c9ce4f2a8ee4574f9d55d5893d472af94]{
 -- Page 436, Offset 32919.
 -- だからこそ、ヤツの目は恐怖だけを映していた。
-For that reason, his eyes reflect only fear.
+That is why the only thing reflected in his eyes is fear.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:553]: that's

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 7]
```diff
@@ -518,42 +565,43 @@ For that reason, his eyes reflect only fear.
...
 -- Page 438, Offset 32928.
 -- 死が視えるのなら―――とても、立ってなんていられない」
-If you could truly see death―――you wouldn't be able to even stand."
+If you could truly see death―――you wouldn't even be able to stand."
 }
 [sha:0dcfa1045baef15addb3b33f3b7b1ba1aa189726]{
 -- Page 438, Offset 32929.
 -- 喩えるのなら、それは月世界。
-To illustrate, it would be like the Lunar World.
+You could liken it to the surface of the moon.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:604]: woah... like the title... moon... so crazy...

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 8]
```diff
@@ -563,93 +611,95 @@ A cold wasteland where everything is dead.
...
 -- Page 438, Offset 32931.
 -- ……物事の“死”が視えるという事は、すべてがあやふやで脆い世界に投げ込まれる事だ。
-...To see the 'death' of things is to be thrown into a world where everything is vague and brittle.
+...To see the 'death' of things is to be thrown into a world where everything is brittle and uncertain.
 }
 [sha:6cb4d574f6fe8e6650fe23534a014318b35e02a8]{
 -- Page 438, Offset 32932.
 -- 地面は無いに等しく、空は今にも落ちてきそうなほど、弱々しい。
-So frail that the ground is like nothingness, and the skies could fall at any moment.
+A world so frail that the ground is almost insubstantial, and the sky could fall at any moment.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:619]: I see where you're getting "frail" from but I think "fragile" would be more natural

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 9]
```diff
@@ -563,93 +611,95 @@ A cold wasteland where everything is dead.
...
 -- ……それはそうだろう。
-...That's only natural.
+...It's to be expected.
 }
 [sha:70a73014a18bd76f15ced42af642d83a54702298]{
 -- Page 439, Offset 32936.
 -- 俺の独唱をあいつは一ミリだって理解できない。
-// 独唱 lmao wtf
-He isn't able to comprehend even a single milimeter of my solo performance.
+He can't even begin to understand my monologue.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:639]: image

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 10]
```diff
@@ -679,17 +731,17 @@ As such, there's no need for me to reconfirm.
...
 -- Page 444, Offset 32954. Glued.
 -- これが、モノを殺すということだ」
- This is what it truly means to kill."
+ This is what it truly means to kill something."
 }
 [sha:6bc390bd37c3166e361874d03996f7c14de23fef]{
 -- Page 444, Offset 32955.
 -- この一刺しの方が、何倍も簡潔だ。
-The swipe is many times more concise than ever before.
+I don't have to rethink anything. I just cut.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:739]: Can you explain to me where "rethink" is coming from? This reads to me like lit. "this 'one stab' [of the point of death] method is many times more succinct". I don't really see where your version is coming from

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 11]
```diff
@@ -226,17 +252,19 @@ And yet―――even though I no longer have the power to stand up, I have somet
...
 -- 「そうだ。君は私の敵にはなりえない。
-"That's right. I don't see you as my enemy.
+"That's right. It's impossible for you to be my opponent.
 }
 [sha:577a58e0223a7ac00b305c4940e049da6987b6da]{
 -- Page 419, Offset 32865.
 -- 性能の差を言っているのではない。殺害が可能か不可能かの喩えでもない。
-It is not a matter of differences in our ability. Neither is it a matter of whether you can kill.
+It's not a matter of the difference in ability between us.
+ Nor is related to the question of whether you could or couldn't kill.
```
Comment (yk1512) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:261]: nor is it

File: script/Arcueid/Day 13/13_00_ARC13_5.txt (hunk 10 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 13/13_00_ARC13_5.txt:739]: The full sentence is 
> それなら無論、ことさら再認するまでもなく、この一刺しの方が、何倍も簡潔だ。

I'd originally read it as like "I don't need to readjust my plans (再認する)" but yeah looking at it again not sure where I got that. I think I was confused by what exactly 再認 was doing. Seems like it's really more like
> If that [Roa not fast] is the case, then it naturally follows, without even having to think about it, that my one slice of the knife is many many times faster [than him].

and so I'd probably update the TL along the lines of
> If that's the case, then the outcome is already a foregone conclusion.
> 「speechifying」
> I can plunge my knife down far faster than he can reach me.

File: script/Arcueid/Day 13/13_00_ARC13_5.txt (hunk 9 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 13/13_00_ARC13_5.txt:639]: Is :jimface: good or bad

File: script/Arcueid/Day 13/13_00_ARC13_5.txt (hunk 7 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 13/13_00_ARC13_5.txt:604]: I think that this is the chapter title (Lunar World) but nobody _says_ that so I think people will just have to make the connection on their own like good little literate media consumers

File: script/Arcueid/Day 13/13_00_ARC13_5.txt (hunk 9 above)
Comment (Avuxo) [reply script/Arcueid/Day 13/13_00_ARC13_5.txt:639]: it's not your fault. You did a great job with what you were given. But it's bad.

File: script/Arcueid/Day 13/13_00_ARC13_5.txt (hunk 10 above)
Comment (Avuxo) [reply script/Arcueid/Day 13/13_00_ARC13_5.txt:739]: Yeah that new version works for me. I saw there was a comment saying it was continuing and then i dunno, had a power cycle or something? invalidated my mental cache?

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 13/13_00_ARC13_5.txt (hunk 7 above)
Comment (Avuxo) [reply script/Arcueid/Day 13/13_00_ARC13_5.txt:604]: Lol, this wasn't actually made having looked at the old version of the line. I just think this line is very dumb and he uses the god damn moon analogy way too much

**Comment by rschlaikjer** [comment]
Merging this for GPR tomorrow but will circle back if Petrikow enlightens us separately re: 胎児のように

**Pull request merged**